cmd/server.go

```diff
@@ -68,6 +69,11 @@ func serverMain(cmd *cobra.Command, args []string) exitCode {
 		}()
 	}
 
+	// Setup finalizing steps that should be run even if we panic
+	defer func() {
+		journal.Finalize()
```

Comment:
Nit: `Finalize` => `FlushOpenJournals` -- I like the latter better because it makes the behavior explicit in the name.

Reply:
`journal.Finalize` doesn't say the same thing? Maybe `journal.FlushOpen`? `journal.FlushOpenJournals` feels a little redundant.

Reply:
`Finalize` seemed a bit too abstract to me, like we're doing more things there than just flushing the open journals (which we're not).

`journal.FlushOpen` is fine too.

Reply:
What about `journal.CloseAll`? Because it's both flushing and closing.

Reply:
Yeah, `journal.CloseAll` sounds good.

Reply:
Updated.